packages/gatsby-source-filesystem/package.json

```diff
@@ -14,7 +14,7 @@
     "file-type": "^12.4.0",
     "fs-extra": "^8.1.0",
     "gatsby-core-utils": "^1.0.26",
-    "got": "^7.1.0",
+    "got": "^9.6.0",
```

Comment:
We might need to wait until we trully drop Node 8 - we added deprecation note in https://github.com/gatsbyjs/gatsby/pull/20466, so we need to keep tracking node version usage

Reply:
Aha. In that case we could move to got@10, which drops 8 support. I *think* we can make do for now with got@8, which has the progress events that we need. I'll take a look on Monday

Reply:
Ah, yeah - was just writing about `got@8` ;)